### PR TITLE
Add camel-joor wrapper

### PR DESCRIPTION
**Pull request by lopushen**

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -1160,6 +1160,13 @@
         <bundle dependency='true'>wrap:mvn:com.bazaarvoice.jolt/jolt-core/${jolt-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-jolt/${project.version}</bundle>
     </feature>
+    <feature name='camel-joor' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <!-- wrap protocol needed because joor has a Require-Capability: osgi.ee;filter:="(osgi.ee=UNKNOWN) in its manifest.
+         The wrap does not support & , so it's not possible to add version to the filter -->
+        <bundle dependency='true'>wrap:mvn:org.jooq/joor/${joor-version}$overwrite=merge&amp;Require-Capability=osgi.ee;filter:="(osgi.ee=JavaSE)"</bundle>
```
Comment (lopushen) [inline features/src/main/feature/camel-features.xml]: used the same solution as in camel-csimple-joor

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -1160,6 +1160,13 @@
         <bundle dependency='true'>wrap:mvn:com.bazaarvoice.jolt/jolt-core/${jolt-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-jolt/${project.version}</bundle>
     </feature>
+    <feature name='camel-joor' version='${project.version}' start-level='50'>
```
Comment (f2par0) [inline features/src/main/feature/camel-features.xml]: should be after camel-jooq to respect alphabetical order

Comment (lopushen) [reply features/src/main/feature/camel-features.xml]: I gotta revise the alphabet:D Fixed, thanks

**Review by f2par0 — APPROVED** [review]
- no conflict :heavy_check_mark: 
- compilation works :heavy_check_mark: 
- no unnecessary usage of wrap protocol :heavy_check_mark: 
- no unnecessary dependencies :heavy_check_mark: 
- no hardcoded version if it's in the parent pom :heavy_check_mark: 
- feature is placed in alphabetical order in the file camel-features.xml  :heavy_check_mark: 
- usage of dependent features (netty, jackson, jakarta ...) :heavy_check_mark:

**Review by essobedo — APPROVED** [review]

**Pull request merged**